test/uk/gov/hmrc/plasticpackagingtax/returns/views/StartPageViewSpec.scala

```diff
@@ -0,0 +1,148 @@
+        messages("startPage.title")
+      )
+      startPage.render(request, messages).select("title").text() must include(
+        messages("startPage.title")
+      )
+    }
+
+    "display title" in {
+
+      view.select("title").text() must include(messages("startPage.title"))
+    }
+
+    "display section header" in {
+
+      view.getElementById("section-header") must containMessage("startPage.title.sectionHeader")
+    }
+
+    "display header" in {
+
+      view.getElementById("title") must containMessage("startPage.title")
+    }
+
+    "display 'Information you need' section" in {
+
+      view.getElementById("section-1") must containMessage("startPage.informationYouNeed.header")
+      view.getElementsByClass("govuk-body").get(0) must containMessage(
+        "startPage.informationYouNeed.body"
+      )
+      view.getElementsByClass("govuk-body").get(1) must containMessage(
+        "startPage.informationYouNeed.listItems.header"
+      )
+      view.getElementsByClass("dashed-list-item").get(0) must containMessage(
+        "startPage.informationYouNeed.listItem.1"
+      )
+      view.getElementsByClass("dashed-list-item").get(1) must containMessage(
+        "startPage.informationYouNeed.listItem.2"
+      )
+      view.getElementsByClass("dashed-list-item").get(2) must containMessage(
+        "startPage.informationYouNeed.listItem.3"
+      )
+      view.getElementsByClass("dashed-list-item").get(3) must containMessage(
+        "startPage.informationYouNeed.listItem.4"
+      )
+      view.getElementsByClass("dashed-list-item").get(4) must containMessage(
+        "startPage.informationYouNeed.listItem.5"
+      )
+    }
+
+    "display 'What is liable?" in {
+
+      view.getElementById("section-2") must containMessage("startPage.whatIsLiable.header")
+      view.getElementsByClass("govuk-body").get(2) must containMessage(
+        "startPage.whatIsLiable.body.1"
+      )
+      view.getElementsByClass("govuk-body").get(3) must containMessage(
+        "startPage.whatIsLiable.body.2"
+      )
+      view.getElementsByClass("govuk-body").get(4) must containMessage(
+        "startPage.whatIsLiable.listItems.header"
+      )
+      view.getElementsByClass("dashed-list-item").get(5) must containMessage(
+        "startPage.whatIsLiable.listItem.1"
+      )
+      view.getElementsByClass("dashed-list-item").get(6) must containMessage(
+        "startPage.whatIsLiable.listItem.2"
+      )
+      view.getElementsByClass("dashed-list-item").get(7) must containMessage(
+        "startPage.whatIsLiable.listItem.3"
+      )
+      view.getElementsByClass("govuk-body").get(5).text() must include(
+        messages("startPage.whatIsLiable.body.3", messages("startPage.whatIsLiable.body.3.link"))
+      )
+    }
+
+    "display 'Start now' button" in {
+
+      view.getElementsByClass("govuk-button").first() must containMessage("startPage.buttonName")
+      view.getElementsByClass("govuk-button").first() must haveHref(
```

Comment:
Do we need a similar test for the `Plastic Packaging Tax guidance` link? i.e. that it's a href?

Reply:
Test added